apps/backend/test/mocks/user.ts

```diff
@@ -1,6 +1,20 @@
-import { UserType } from '@animavita/models';
+import { UserEntity } from 'src/user/repositories/user-repository.interface';
 
-export const userMock: UserType = {
+export const userMock: UserEntity = {
+  name: 'Grosbilda',
+  email: 'grosbilda@email.com',
+  password: 'quesenharuimgrosbilda',
+  location: {
+    latitude: 10.002322,
+    longitude: -40.343569,
+  },
+  photoUri: 'http://www.google.com/',
+  id: '123',
+  createdAt: '',
+  updatedAt: '',
+};
```

Comment:
Also out of the scope of this PR but would be nice to set up [Faker](https://fakerjs.dev/) for that